### PR TITLE
Verify the correct use of "IP address" in the whole documentation

### PR DESCRIPTION
**Pull request by Arcapas**
Hi team,

## Description
This issue aims to verify the correct use of "IP address" in the whole documentation:
142271924-0a5ddbcb-8705-43c7-aad6-51a16c11cbe3

Activities of the issue
----------------------------
- [x] Create a branch: `4601_4.2_verify_ip_address_text`
- [x] Edit the text verifying the correct use of "IP address"
- [x] Make a complete review of the whole documentation
- [x] Perform a revision of all the changes
- [x] Ask content team for review
- [x] This PR closes issue #4601 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Regards,
Damian

### CODE REVIEW

File: source/user-manual/configuring-cluster/index.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ Deploying a Wazuh cluster
 
         - :ref:`node_type <cluster_node_type>`: Set the node type (master/worker)
         - :ref:`port <cluster_port>`: Destination port for cluster communication
-        - :ref:`bind_addr <cluster_bind_addr>`: IP where this node is listening to (0.0.0.0 any IP)
+        - :ref:`bind_addr <cluster_bind_addr>`: IP addresses where this node is listening to (0.0.0.0 any IP)
```
Comment (luiscontrerasdo) [inline source/user-manual/configuring-cluster/index.rst:38]: Para este caso entiendo que debe ser en singular IP address, ya que hace referencia a una IP.

Comment (Arcapas) [reply source/user-manual/configuring-cluster/index.rst:38]: Estoy de acuerdo. Lo cambio.

Comment (Arcapas) [inline source/user-manual/configuring-cluster/index.rst:38]:
```suggestion
        - :ref:`bind_addr <cluster_bind_addr>`: IP address where this node is listening to (0.0.0.0 any IP)
```

**Review by luiscontrerasdo — APPROVED** [review]
LGTM

**Review by javimed — APPROVED** [review]

**Review by MarVergara — APPROVED** [review]
LGTM

**Pull request merged**